qiskit/tools/parallel.py

```diff
@@ -72,7 +72,7 @@ def get_platform_parallel_default():
         parallel_default = False
     # On macOS default false on Python >=3.8
     elif sys.platform == "darwin":
-        if sys.version_info[0] == 3 and sys.version_info[1] >= 8:
+        if sys.version_info >= (3, 8) and sys.version_info < (4,):
```

Comment:
`(3, 8) <= sys.version_info < (4,)` is also a valid way of spelling this, though tbh I don't think it's super important to defensively protect against a hypothetical Python 4.

Reply:
the chained inequality is more clear. I only did this because there was an existing test involving a hypothetical Python 10 (!)

Reply:
Suggestion implemented in 03f4b6bd2db699a97d169843f2859505cb6515e5